docs/en/latest/FAQ.md

```diff
@@ -371,3 +371,34 @@ You may have to install OPENSSL in your system and/or pass OPENSSL_DIR or OPENSS
 Example: luarocks install luasec OPENSSL_DIR=/usr/local
 make: *** [deps] Error 1
 ```
+
+## How to access APISIX Dashboard through APISIX proxy
+
+1. Keep APISIX and Admin API using different ports (or disable Admin API), for example, do the following configuration in `conf/config.yaml`.
+
+The Admin API use a separate port 9180:
+
+```yaml
+apisix:
+  port_admin: 9180            # use a separate port
+```
+
+2. Add proxy route of APISIX Dashboard:
+
+```shell
+curl -i http://127.0.0.1:9180/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uris":[ "/*" ],
+    "name":"apisix_proxy_dashboard",
+    "upstream":{
+        "nodes":[
+            {
+                "host":"127.0.0.1",
```

Comment:
Should add a note to tell the user that assuming we have a Dashboard service listening on `127.0.0.1:9000`.

Reply:
updated.